data/competencies.yml

```
@@ -470,9 +470,16 @@
   domain: null
 
 - id: 9e1a0313-f602-4786-97c6-d4d49506f0bb
-  summary: Does not unnecessarily duplicate work of other teams
-  examples: []
-  description: null
+  summary: Where appropriate, builds on other teams' work to solve problems
+  examples:
+    - Uses origami components to style a web page
+    - Records information about technical infrastructure in Biz Ops 
+  description: |
```

We should use [folded style](https://yaml.org/spec/1.2/spec.html#id2796251) for description blocks, so that line breaks are not preserved
```suggestion
  description: >
```

---

I did not know that was the distinction

Happy to split the description into a separate PR. I think they can be useful in a lot of cases - possibly the majority as we move up the seniority scale.

---

Yeah I'm definitely finding that I've forgotten a lot of the context and decision-making process behind a few of the competencies. Writing it down _somewhere_ would be good.

Agree on splitting 👍